subsys/net/lib/coap/coap.c

```diff
@@ -913,6 +913,16 @@ static int method_from_code(const struct coap_resource *resource,
 	}
 }
 
+static bool is_empty_emtpy_message(const struct coap_packet *cpkt)
+{
+	uint8_t code = __coap_header_get_code(cpkt);
+
+	if (code == COAP_CODE_EMPTY) {
+		return true;
+	}
+	return false;
+}
```

Comment:
Typo in function name, and can be simplified:
```suggestion
static inline bool is_empty_message(const struct coap_packet *cpkt)
{
	return __coap_header_get_code(cpkt) == COAP_CODE_EMPTY;
}
```

Reply:
ah let me fix.